examples/SampleApp/src/hooks/useStreamChatTheme.ts

```diff
@@ -57,6 +58,16 @@ const getChatStyle = (colorScheme: ColorSchemeName): DeepPartial<Theme> => ({
           white_smoke: '#F2F2F2',
           white_snow: '#FCFCFC',
         },
+  ...(colorScheme === 'dark'
+    ? {
+        messageSimple: {
```

Comment:
Added mistakenly?

Reply:
Nope, this is something that was broken in the `SampleApp` and dark mode was not working at all. Took the opportunity to fix it.